theme/static/js/hs-vue/funding-agency-app.js

```diff
@@ -224,14 +224,25 @@ let fundingAgenciesApp = new Vue({
       this.mode = "Edit";
       this.notifications = [];
       this.filteredWords = [];
+      
       const editingFundingAgency = this.fundingAgencies.filter((agency) => {
         return agency.agency_id == id;
       })[0];
+      
       this.currentlyEditing = { ...editingFundingAgency };
       this.startedEditing = { ...editingFundingAgency };
       this.agencyNameInput = this.currentlyEditing.agency_name;
       // open source bug https://github.com/alexurquhart/vue-bootstrap-typeahead/issues/19
       this.$refs.agencyNameInput.inputValue = this.currentlyEditing.agency_name;
+    
+      // Only check funder names when editing the resource
+      if (this.selfAccessLevel === "owner" && this.resPublished) {
+        this.checkFunderNamesExistInCrossref(this.fundingAgencies).then(() => {
+          if (this.unmatchedFunders.length > 0) {
+            this.showFundersAlert(); // Show alert if unmatched funders are found
+          }
+        });
+      } 
```

Comment:
this check will only be run when `openEditModal` is called. This is not the same as when the resource is in "edit mode" -> `openEditModal` is a function within this Vue app that is called when one clicks the button to edit a single funder
![image](https://github.com/user-attachments/assets/a02ea63d-a0e2-41a1-9d31-7265591804c2)

Reply:
we discussed this, marking resolved